client/web/src/global/CoolCodeIntel.tsx

```diff
@@ -369,18 +424,38 @@ const CollapsibleLocationList: React.FunctionComponent<{
                             <ChevronRightIcon className="icon-inline" aria-label="Expand" />
                         )}{' '}
                         {capitalize(props.name)}
+                        <Badge pill={true} variant="secondary" className="ml-2">
+                            {props.locations.length}
+                            {props.hasMore && '+'}
+                        </Badge>
                     </h4>
                 </Button>
             </CardHeader>
 
             <Collapse id="references" isOpen={isOpen}>
                 {props.locations.length > 0 ? (
-                    <LocationsList
-                        locations={props.locations}
-                        activeLocation={props.activeLocation}
-                        setActiveLocation={props.setActiveLocation}
-                        filter={props.filter}
-                    />
+                    <>
+                        <LocationsList
+                            locations={props.locations}
+                            activeLocation={props.activeLocation}
+                            setActiveLocation={props.setActiveLocation}
+                            filter={props.filter}
+                        />
+                        {props.hasMore &&
+                            props.fetchMore !== undefined &&
+                            (props.loadingMore ? (
+                                <div className="text-center mb-1">
+                                    <em>Loading more {props.name}...</em>
+                                    <LoadingSpinner inline={true} />
+                                </div>
+                            ) : (
+                                <div className="text-center mb-1">
+                                    <Button variant="secondary" onClick={fetchMore}>
```

Comment:
```suggestion
                                    <Button variant="secondary" onClick={props.fetchMore}>
```

We can delete the callback wrapper, as we're already checking that it's not undefined in the ternary, but also if it was undefined it just wouldn't apply the `onClick`.